### ISSUE TITLE
Choose file button in add media flow not setting correct focus

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19817]
## User Story or Problem Statement
In the media upload flow:
As a screen reader user or keyboard only user when I choose a file to upload or remove an upload, my focus should move to the equivalent element.

Current state:
- As a screen reader user when I choose a file to upload, my focus gets changed to the top of the main element and when I press the remove button my focus gets set to above the Image dropdown
- As a keyboard only user when I choose a file to upload, my focus goes to above the Image dropdown but, when I press the remove button it sets my focus to the top of the page.

## Description or Additional Context 
Not sure why the focus changes based on if you're using a screen reader or a keyboard, but the focus should always go to the choose file button when removing an image and the image name after uploading the image.

## Steps for Implementation
The focus should always go back to the equivalent element from where it last was in the upload and remove process.

## Acceptance Criteria
- [ ] The keyboard only user focus should be set to the file name after upload and to the Choose file button after remove
- [ ] The screen reader user focus should be set to the file name after upload and to the Choose file button after remove
- [ ] Requires accessibility review
